.github/workflows/docker-bases.yml

```diff
@@ -17,7 +17,7 @@ on:
         description: 'Build compiler bases'
   schedule:
     # Run once a week
-    - cron: "0 13 * * 1"
+    - cron: "0 0 1 * *"
```

Comment:
Note: Once a month should be enough overkill to do once a week.

Reply:
Then you should update the comment above